lib/Gateway.js

```diff
@@ -47,20 +54,36 @@ function loadCustomDevices (devices) {
   )
 }
 
+var lastCustomDevicesLoad = 0
+function tryLoadCustomDevices (file, first) {
+  if (Date.now() - lastCustomDevicesLoad < 1000) {
```

Comment:
Is this to prevent the double call from watch?

Reply:
This is to maintain the previous `lock` behavior. Personally I don't think the double call is that problematic, but I didn't want to change the behavior.